config/zones.json

```diff
@@ -328,7 +328,7 @@
     "flag_file_name": "au.png",
     "timezone": null,
     "parsers": {
-      "production": "AUS_TAS_KI.fetch_production"
+      "production": "adjenti.fetch_production"
```

Comment:
It's spelled "ajenti"

Alternatively, you could also name this after Hydro Tasmania

Reply:
damn me. Thanks for the catch. 

I'm going to stay with ajenti as they're the Hydro Tasmania contractor/subsidiary for these project and I think they could power projects for other operators (see [who are we ? from their website](https://adms.ajenti.com.au/Info/projects.htm))